Docs pm 2.9 dh (#143)
mulesoft/docs-partner-manager: modules/ROOT/pages/delete-message-flows.adoc

> == Delete Message Flows

Reviewer: ```suggestion
== Delete a Message Flow
```

> == Delete a Message Flow

Reviewer: here

> You can delete message flows if they no longer serves their purpose, for example, if your relationship with your partner changes. You can delete message flows from a sandbox or production environment.

Reviewer: ```suggestion
You can delete message flows that no longer serve their purpose, for example, if your relationship with your partner changes. You can delete message flows from a sandbox or production environment.
```

> Deleting a message automatically undeploys the flow. If you Delete Message Flows before it is deployed, the Mule apps are not affected. [*Vijay. What Mule apps is this referring to?*]

Reviewer: ```suggestion
Deleting a message from a flow automatically undeploys the flow, unless that flow has not yet been deployed. [*Vijay. What Mule apps is this referring to?*]
```
Do we need to document this here and in this way? The topic is deleting message flows; we do not want to suggest deleting a message to do so, do we? That sounds like an unintentional action and result.

[NOTE]
For on-premises installations, you must xref:deploy-message-flows#undeploy-message-flows[undeploy a message flow] before you can delete it.

To delete message flows:

. In the navigation menu, select *Message Flows*.
. Select the message flow to delete and click *Delete* in the upper right of the screen.

> Previous transmissions that were created as a result of a message being processed by the message flow you deleted have the following behavior:

Reviewer: Possibility
```suggestion
Transmissions already created as a result of a message being processed by the message flow you deleted:
```

> * The transmissions continue to stay in the transmission records within the *Activity* page; however, the records indicate that the corresponding message flow used to process the transmission is now deleted.

Reviewer: ```suggestion
* Remain in the transmission records within the *Activity* page with a notation that the corresponding message flow is now deleted.
```

* [*???*]These activities are converted into read-only mode, so no links to the message flow work.

> * Only links to the stored payloads (such as the original B2B message, any TA1 acknowledgments, the transformed payload, on so on) continue to work.

Reviewer: * Honor links to stored payloads only, such as the original B2B message, any TA1 acknowledgments, and the transformed payload.

Reviewer: Not sure that a bullet list is the best way to say that those transmissions already sent by a message flow that is subsequently deleted remain recorded in the *Activity* page, and links to their output remain active in stored payloads.
